helix-core/src/main/java/org/apache/helix/monitoring/mbeans/dynamicMBeans/DynamicMBeanProvider.java

```diff
@@ -223,4 +225,13 @@ protected void incrementSimpleDynamicMetric(SimpleDynamicMetric<Long> metric) {
   protected void incrementSimpleDynamicMetric(SimpleDynamicMetric<Long> metric, long value) {
     metric.updateValue(metric.getValue() + value);
   }
+
+  /**
+   * Return the interval length for the underlying reservoir used by the MBean metric configured
+   * in the system env variables. If not found, use default value.
+   */
+  protected Long getResetIntervalInMs() {
```

Comment:
use primitive type long

Reply:
This is from another PR. We should fix it after this gets merged with master. Otherwise, there will be a merge conflict.